cvat/apps/documentation/user_guide.md

```diff
@@ -361,11 +435,24 @@ Go to specified frame. Press ``~`` to highlight element.
 ---
 __Open Menu__ button
 
-It is the main menu for the annotation tool. It can be used to download, upload and remove annotations.
+It is the main menu for the annotation tool. It can be used to download, upload and remove annotation.
 
 ![](static/documentation/images/image051.jpg)
 
-As well it shows statistics about the current task, such as:
+Button assignment:
+
+- ``Open Task`` — open task in cvat dashboard
+- ``RunReID Merge`` — combines unbound tracks generated by using autoannotation into one object. [Read more](https://github.com/opencv/cvat/tree/develop/cvat/apps/reid)
+- ``Dump Annotation`` — download an annotation file from the task 
+- ``Upload Annotation`` — uploading an annotation file to the task
+- ``Remove Annotation`` — remove annotation from current task
```

Comment:
Need to pay attention to "annotation" word. In many cases it is better to use plural form. If you talk about "annotated objects" it is better to say "annotations". If you talk about an "annnotation process" it is possible to say "annotation".